gui/public/i18n/ru/translation.ftl

```diff
@@ -0,0 +1,542 @@
+settings-general-tracker_mechanics-drift_compensation = Компенсация дрифта
+# This cares about multilines
+settings-general-tracker_mechanics-drift_compensation-description =
+    Компенсирует дрейф по рысканию IMU путем применения обратного вращения.
+    Измените размер компенсации и до скольких сбросов учитывается.
+settings-general-tracker_mechanics-drift_compensation-enabled-label = Компенсация дрифта
+settings-general-tracker_mechanics-drift_compensation-amount-label = Кол-во компенсации
+settings-general-tracker_mechanics-drift_compensation-max_resets-label = Используйте до x последних сбросов
+
+## FK/Tracking settings
+settings-general-fk_settings = Настройки трекеров
+settings-general-fk_settings-leg_tweak = Починка ног
+settings-general-fk_settings-leg_tweak-description = Зажим для пола может уменьшить или даже устранить сцепление с полом, но может вызвать проблемы, когда вы стоите на коленях. Катание на коньках-коррекция корректирует катание на коньках, но может снизить точность в определенных моделях движений.
+# Floor clip:
+# why the name - came from the idea of noclip in video games, but is the opposite where clipping to the floor is a desired feature
+# definition - Prevents the foot trackers from going lower than they where when a reset was performed
+settings-general-fk_settings-leg_tweak-floor_clip = Клип через пол
+# Skating correction:
+# why the name - without this enabled the feet will often slide across the ground as if your skating across the ground,
+# since this largely prevents this it corrects for it hence skating correction (note this may be renamed to sliding correction)
+# definition - Guesses when each foot is in contact with the ground and uses that information to improve tracking
+settings-general-fk_settings-leg_tweak-skating_correction = Коррекция катания на коньках
+settings-general-fk_settings-leg_tweak-skating_correction-amount = Сила коррекция катания на коньках
+settings-general-fk_settings-arm_fk = Трекинг руки
+settings-general-fk_settings-arm_fk-description = Измените способ отслеживания рук.
+settings-general-fk_settings-arm_fk-force_arms = Руки от HMD
+settings-general-fk_settings-skeleton_settings = Настройки скелета
+settings-general-fk_settings-skeleton_settings-description = Включите или выключите настройки скелета. Рекомендуется оставить их включенными.
+settings-general-fk_settings-skeleton_settings-extended_spine = Удлененный позвоночник
+settings-general-fk_settings-skeleton_settings-extended_pelvis = Расширенный таз
+settings-general-fk_settings-skeleton_settings-extended_knees = Вытянутое колено
+settings-general-fk_settings-vive_emulation-title = Эмуляция Vive
+settings-general-fk_settings-vive_emulation-description = Имитируйте проблемы с отслеживанием талии, которые возникают у трекеров Vive. Это шутка, и она ухудшает отслеживание.
+settings-general-fk_settings-vive_emulation-label = Включить эмуляцию Vive
+
+## Gesture control settings (tracker tapping)
```

Comment:
This section isn't translated? If you're unsure what it's for feel free to ask any questions in https://discord.com/invite/slimevr , see the dev-forum , and the "Translation - General" post :)